serving/samples/helloworld-haskell/README.md

```diff
@@ -0,0 +1,198 @@
+# Hello World - Haskell sample
+
+A simple web app written in Haskell that you can use for testing.
+It reads in an env variable `TARGET` and prints "Hello World: ${TARGET}!". If
+TARGET is not specified, it will use "NOT SPECIFIED" as the TARGET.
+
+## Prerequisites
+
+* A Kubernetes cluster with Knative installed. Follow the
+  [installation instructions](https://github.com/knative/docs/blob/master/install/README.md) if you need
+  to create one.
+* [Docker](https://www.docker.com) installed and running on your local machine,
+  and a Docker Hub account configured (we'll use it for a container registry).
+
+## Steps to recreate the sample code
```

Comment:
Change header to "Recreating the sample code"

Reply:
I will work on this today. Thanks for the inputs.